### ISSUE TITLE
[Victory Axis] Forced to specify minimum and maximum for domain prop

### ISSUE THREAD

**Issue by OwenFlood** [FormidableLabs/victory#530]
In my app the chart will sometimes have no data in it, and when this happens the `y` axis looks like this:
<img width="362" alt="victory chart - empty example" src="https://cloud.githubusercontent.com/assets/17835216/24126715/ae2146bc-0d8c-11e7-9ddc-2f5101c8f4db.png">

I tried using the `domain` prop for the axis, but since both the minimum and maximum props are required I was forced to calculate the highest bar and add like 1000 for space, then pass that variable to the `domain` prop.

I think that it would be helpful to be able to specify either on their own for a cleaner looking graph :)

**Comment by boygirl** [comment]
@OwenFlood The idea of having a domain take a single value doesn't really make sense, because it _needs_ a min and a max to work properly. 

You might check out the [domainPadding](https://formidable.com/open-source/victory/docs/victory-chart#domainpadding) prop on VictoryChart, or try specifying [tickValues](https://formidable.com/open-source/victory/docs/victory-axis#tickvalues) on VictoryAxis.

I'm going to close this issue, but I would be happy to re-open it if you can tell me a little more about what you think your axis should look like without data. What should the defaults be, where should they come from?

You can also message me in the [gitter channel](https://gitter.im/FormidableLabs/victory) for questions and help.

Thanks!

**Comment by OwenFlood** [comment]
Hi @boygirl, what I was trying to do is have a minimum value for y-axis because when the chart isn't populated it shows the values in thousands still (like in the picture I linked). I originally tried using the `domain` prop because it was the only one referencing a minimum tick value, but since I can't know what the maximum of the chart will be I run into problems.

TLDR: I think that my problem is from setting the y-axis' increments to thousands, so when the value is 0, the y-axis shows 0.002 instead of 2. It's definitely an edge case so I can understand if it's not the most pressing matter 😛

**Comment by boygirl** [comment]
if you know what you want your `tickValues` to be in advance, you can set that prop on VictoryAxis, and get roughly the result you are looking for. If you set `tickValues` on `VictoryAxis` and _don't_ set a `domain` prop on `VictoryChart` the calculated domain will take your tickValues and your data into account. This should work out alright if your data is predictable and doesn't include extreme values.

Unfortunately the idea of being able to specify only a domain minimum or only a domain maximum is not feasible.
